core/modules/css/cssLib.i

```diff
@@ -47,7 +53,118 @@ namespace std {
     %template(StringVector) vector<string>;
 };
 
+%include "sql/SqlErrorObject.h"
+
+%{
+    void setPythonException(const std::string& exceptionStr, const lsst::qserv::css::CssError& ex) {
+        PyObject* module = PyImport_ImportModule("lsst.qserv.css");
+        if (!module)
+            throw;
```

Comment:
My SWIG is very rusty, what happens here if you throw C++ exception here, is it caught by SWIG layer or does it crash whole thing?